news/5624.bugfix

```diff
@@ -0,0 +1 @@
+Fix installing a Git ref for installs after the first.
```

Comment:
I think this could be more clear, it was not obvious to me what this change was about until I actually looked at the code. Perhaps "Allow a git ref to be installed over an existing non-ref install"

Reply:
Right, so maybe "Allow a git ref to be installed over an existing installation" then?

That said I think _most_ casual users are not going to understand this change regardless so perhaps it's fine as is. 🙂